framework/src/st_things/Kconfig

```diff
@@ -17,6 +17,9 @@ config ST_THINGS
 	depends on FS_ROMFS
 	---help---
 		SmartThings Things Management for applications to use and control devices
+	depends on NETUTILS_NTPCLIENT
```

Comment:
1. Your source codes look like not mandatory. It looks option. Why do you set it as ```depends on```?
2. Move this before above ```---help---``` and remove below help.